cmd/agent/app/integrations.go

```diff
@@ -290,13 +292,38 @@ func install(cmd *cobra.Command, args []string) error {
 		// Specific case when installing from locally available wheel
 		// No compatibility verifications are performed, just install the wheel (with --no-deps still)
 		// Verify that the wheel depends on `datadog_checks_base` to decide if it's an agent check or not
+		wheelPath := args[0]
+
 		fmt.Println(disclaimer)
-		if ok, err := validateBaseDependency(args[0], nil); err != nil {
-			return fmt.Errorf("error while reading the wheel %s: %v", args[0], err)
+		if ok, err := validateBaseDependency(wheelPath, nil); err != nil {
+			return fmt.Errorf("error while reading the wheel %s: %v", wheelPath, err)
 		} else if !ok {
-			return fmt.Errorf("the wheel %s is not an agent check, it will not be installed", args[0])
+			return fmt.Errorf("the wheel %s is not an agent check, it will not be installed", wheelPath)
+		}
+
+		// Parse the package name from metadata contained within the zip file
+		integration, err := parseWheelPackageName(wheelPath)
+		if err != nil {
+			return err
+		}
+		integration = normalizePackageName(strings.TrimSpace(integration))
+
+		// Install the wheel
+		if err := pip(append(pipArgs, wheelPath), os.Stdout, os.Stderr); err != nil {
+			return fmt.Errorf("error installing wheel %s: %v", wheelPath, err)
 		}
-		return pip(append(pipArgs, args[0]), os.Stdout, os.Stderr)
+
+		// Move configuration files
+		if err := moveConfigurationFilesOf(integration); err != nil {
+			fmt.Printf("Installed %s from %s", integration, wheelPath)
```

Comment:
why not use the `log` package here ?

Reply:
We want to display things in the console afaik